examples/slurm/start-slurm.sh

```diff
@@ -73,9 +73,6 @@ export LIBCUDF_CUFILE_POLICY=OFF
 # End easy customization
 # =================================================================
 
-mkdir -p $LOGDIR
-mkdir -p $PROFILESDIR
```

Comment:
And this outside, do the paths line up ?

Reply:
See other comment.